core/storage/collection/gae_models.py

```diff
@@ -142,13 +153,19 @@ def _trusted_commit(
 class CollectionRightsSnapshotMetadataModel(
         base_models.BaseSnapshotMetadataModel):
     """Storage model for the metadata for a collection rights snapshot."""
-    pass
+    @staticmethod
+    def get_export_policy():
+        """Model does not contain user data."""
```

Comment:
Don't the rights models contain user IDs for who can edit the collection, etc.?

Reply:
Isn't this entire model a pass? This model doesn't seem to be defined in any way, so I'm not sure how it could have user data.

Reply:
Look at the class it inherits from and how instances of that class are constructed -- I think either this or the snapshot content model do contain user data. /cc @vojtechjelinek

Reply:
@seanlip per the suggestion you made about looking at the has_reference_to_user_id() I did not change this, since has_reference_to_user_id() is not even defined for this model. Please let me know if you think that there are some other changes that I have to make.